routes/articles.js

```diff
@@ -9,26 +9,45 @@ router.post('/', async (req,res) => {
         title : req.body.title,
         description :  req.body.description,
         author : req.body.author,
-        tags :  req.body.tags
+        tags :  req.body.tags,
+        time : req.body.time
```

Comment:
isnt't it date.now() 
then why ?

Reply:
if the user wants to pass a custom date they can do that, but if it is not passed the ```req.body.time``` will be undefined and thus it will our .save function will set it as```date.now()```.